lib/cisco_node_utils/yum.rb

```diff
@@ -20,8 +20,15 @@
 require_relative 'node_util'
 
 module Cisco
-  # This Yum class provides cisco package management functions through nxapi.
+  # This Yum class provides cisco package management functions.
   class Yum < NodeUtil
+    EXEC_IN_DEFAULT_NS = 'ip netns exec default'
+
+    # This function accepts name of the rpm package and returns a match group.
+    # Match group when platform is ios_xr :
+    #   1. package name, 2. package version, 3. os version, 4. platform
+    # Match group when platform is nexus :
+    #   1. package name 2. package version 3. platform
```

Comment:
What about name_arch_regex, which only has two match groups? Perhaps someone from the Nexus side can comment as to the design/usage of this function as it seems very obtuse to me - @mikewiebe, @chrisvanheuveln, @robert-w-gries?